Lib/pathlib.py

```diff
@@ -35,6 +35,12 @@
     _WINERROR_INVALID_NAME,
     _WINERROR_CANT_RESOLVE_FILENAME)
 
+_WIN_RESERVED_NAMES = (
+    {'CON', 'PRN', 'AUX', 'NUL', 'CONIN$', 'CONOUT$'} |
+    {'COM%s' % c for c in '123456789\xb9\xb2\xb3'} |
+    {'LPT%s' % c for c in '123456789\xb9\xb2\xb3'}
```

Comment:
```suggestion
    {f'COM{c}' for c in '123456789\xb9\xb2\xb3'} |
    {f'LPT{c}' for c in '123456789\xb9\xb2\xb3'}
```

I believe f-strings may be used here. According to git blame, original `'COM%d'` was introduced eight years ago in 31119e4, way before *PEP 498 -- Literal String Interpolation*.